plugins.md

```diff
@@ -58,6 +58,7 @@ Name | Description | Stars
 [pod-logs](https://github.com/danisla/kubefunc) | Display a list of pods to get logs from | ![GitHub stars](https://img.shields.io/github/stars/danisla/kubefunc.svg?label=stars&logo=github)
 [pod-shell](https://github.com/danisla/kubefunc) | Display a list of pods to execute a shell in | ![GitHub stars](https://img.shields.io/github/stars/danisla/kubefunc.svg?label=stars&logo=github)
 [preflight](https://github.com/replicatedhq/troubleshoot) | Executes application preflight tests in a cluster | ![GitHub stars](https://img.shields.io/github/stars/replicatedhq/troubleshoot.svg?label=stars&logo=github)
+[prompt](https://github.com/profefe/kubectl-profefe) | continuous profiling made easy in Kubernetes with profefe. | ![GitHub stars](https://img.shields.io/github/stars/profefe/kube-profefe.svg?label=stars&logo=github)
```

Comment:
404, also consider using cmd/generate-plugin-list from krew to generate this file.